S3/S3.py

```diff
@@ -724,8 +726,10 @@ def object_modify(self, src_uri, dst_uri, extra_headers = None):
 
         headers['x-amz-copy-source'] = encode_to_s3("/%s/%s" % (src_uri.bucket(), self.urlencode_string(src_uri.object())))
         headers['x-amz-metadata-directive'] = "REPLACE"
-
-        # cannot change between standard and reduced redundancy with a REPLACE.
+        if self.config.reduced_redundancy:
```

Comment:
https://docs.aws.amazon.com/AmazonS3/latest/dev/ChgStoClsOfObj.html
says we can't do this in a REPLACE.